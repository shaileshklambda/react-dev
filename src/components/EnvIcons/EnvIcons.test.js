import React from 'react'
import renderer from 'react-test-renderer'
import Envicons from './index.js'

it('Envicons: default', () => {
  const component = renderer.create(<Envicons />)
  const tree = component.toJSON()
  expect(tree).toMatchSnapshot()
})