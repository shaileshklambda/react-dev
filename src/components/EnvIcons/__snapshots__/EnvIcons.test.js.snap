// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Envicons: default 1`] = `
<div
  className="default"
/>
`;
